tools/templates/L10N/IT

```diff
@@ -0,0 +1,411 @@
+#core-prepend       prepend
+#core-print         print
+#core-printf        printf
+#core-proceed       proceed
+#core-prompt        prompt
+#core-push          push
+#core-put           put
+
+# KEY            TRANSLATION
+#core-rand        rand
+#core-redo        redo
+#core-reduce      reduce
+#core-repeated    repeated
+#core-repl        repl
+#core-return      return
+#core-return-rw   return-rw
+#core-reverse     reverse
+#core-rindex      rindex
+#core-rmdir       rmdir
+#core-roll        roll
+#core-rotate      rotate
+#core-round       round
+#core-roundrobin  roundrobin
+#core-run         run
+
+# KEY             TRANSLATION
+#core-samecase     samecase
+#core-samemark     samemark
+#core-samewith     samewith
+#core-say          say
+#core-set          set
+#core-shell        shell
+#core-shift        shift
+#core-sign         sign
+#core-signal       signal
+#core-skip         skip
+#core-sleep        sleep
+#core-sleep-timer  sleep-timer
+#core-sleep-until  sleep-until
+#core-slip         slip
+#core-slurp        slurp
+#core-snip         snip
+#core-snitch       snitch
+#core-so           so
+#core-sort         sort
+#core-splice       splice
+#core-split        split
+#core-sprintf      sprintf
+#core-spurt        spurt
+#core-sqrt         sqrt
+#core-squish       squish
+#core-srand        srand
+#core-subbuf       subbuf
+#core-subbuf-rw    subbuf-rw
+#core-succeed      succeed
+#core-sum          sum
+#core-symlink      symlink
+
+# KEY               TRANSLATION
+#core-tail           tail
+#core-take           take
+#core-take-rw        take-rw
+#core-tc             tc
+#core-tclc           tclc
+#core-trim           trim
+#core-trim-leading   trim-leading
+#core-trim-trailing  trim-trailing
+#core-truncate       truncate
+
+# KEY          TRANSLATION
+#core-uc        uc
+#core-unimatch  unimatch
+#core-uniname   uniname
+#core-uninames  uninames
+#core-uniparse  uniparse
+#core-uniprop   uniprop
+#core-uniprops  uniprops
+#core-unique    unique
+#core-unival    unival
+#core-univals   univals
+#core-unlink    unlink
+#core-unshift   unshift
+
+# KEY        TRANSLATION
+#core-val     val
+#core-value   value
+#core-values  values
+
+# KEY          TRANSLATION
+#core-warn      warn
+#core-wordcase  wordcase
+#core-words     words
+
+# KEY             TRANSLATION
+#infix-after       after
+#infix-and         and
+#infix-andthen     andthen
+#infix-before      before
+#infix-but         but
+#infix-cmp         cmp
+#infix-coll        coll
+#infix-div         div
+#infix-does        does
+#infix-eq          eq
+#infix-ff          ff
+#infix-fff         fff
+#infix-gcd         gcd
+#infix-ge          ge
+#infix-gt          gt
+#infix-le          le
+#infix-lcm         lcm
+#infix-leg         leg
+#infix-lt          lt
+#infix-max         max
+#infix-min         min
+#infix-minmax      minmax
+#infix-mod         mod
+#infix-ne          ne
+#infix-notandthen  notandthen
+#infix-o           o
+#infix-or          or
```

Comment:
There's a collision here, `or` is `o` in Italian. `od` if it's followed by a bowel, sometimes seen as "oppure" meaning "or simply" Any suggestion about what to do here? By default, just leave it as is.

Reply:
`oppure` in that case feels more correct to me.  And then maybe translate `o` with some Italian equivalent of "function composition operator" ?